app/src/main/java/nl/avans/drivioapp/service/DrivioApiService.kt

```diff
@@ -3,10 +3,13 @@ package nl.avans.drivioapp.service
 import com.squareup.moshi.Moshi
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import nl.avans.drivioapp.model.Advertisement
+import nl.avans.drivioapp.model.Reservation
 import retrofit2.Response
 import retrofit2.Retrofit
 import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.http.Body
 import retrofit2.http.GET
+import retrofit2.http.POST
 import retrofit2.http.Path
 
 interface DrivioApiService {
```

Comment:
Heb je nu een service gemaakt voor alle requests aangezien hij DrivioApiService heet?

Gaan we op deze manier werken of maken we voor elke verschillende service een nieuw bestand aan?

Reply:
Nee, dit is een tijdelijke oplossing totdat de repositories zijn goedgekeurd. Dit moet daarna nog omgebouwd worden naar het repository pattern.

Reply:
Ik zag het inderdaad al terug in de repository branch